client/client.go

```diff
@@ -710,6 +812,7 @@ func (c *client) processTSORequests(stream pdpb.PD_TsoClient, dcLocation string,
 		c.finishTSORequest(requests, 0, 0, 0, err)
 		return err
 	}
+	tsoBatchSendLatency.Observe(float64(time.Since(tbc.batchStartTime)))
```

Comment:
Is it differnt with `requestDurationTSO`?

Reply:
`requestDurationTSO` records the elapsed time between RPC sending and receiving. `tsoBatchSendLatency` records the elapsed time between the first TSO request arrives in a batch and the RPC sending.